README.md

```diff
@@ -30,12 +38,52 @@ services:
       - cron.another_task.schedule="*/2 * * * *"
       - cron.another_task.command="/usr/local/bin/another_app_script.sh"
 ```
+This example shows how to schedule multiple cron jobs using cron syntax. The Docker container will run `/usr/local/bin/app_script.sh` every minute and `/usr/local/bin/another_app_script.sh` every two minutes, with logs stored in `/var/log/cron/`.
 
 ### Environment Variables
-
 The `example_compose.yml` file uses several environment variables. Make sure to define these in a `.env` file in the root directory of your project:
+- **`COMPOSE_PROJECT_NAME`**: Specifies the name of the Docker Compose project, allowing the cron to target jobs in that specific project.
+- **`DEBUG`**: Set to `true` to enable detailed output for debugging purposes.
+- **`CRON_LOG_DIR`**: Defines the directory where cron stores log files for executed jobs, defaulting to `/var/log/cron`.
+
+### Volume Mounts
+- **`/var/run/docker.sock`**: This is used to enable the Docker client inside the container to communicate with the Docker daemon running on the host. Be careful when using this as it provides elevated privileges.
+- **`./logs/cron:/var/log/cron`**: Mount a directory to store the cron logs. Make sure this folder has the appropriate permissions.
```

Comment:
I believe that even if we "Modify the Dockerfile to run the container as a non-root user to enhance security and reduce potential risks", the container will check and fix the permissions automatically.